### PR TITLE
Faster CI with cargo-batch

### PR DESCRIPTION
**Pull request by Dirbaio**

### CODE REVIEW

**Review by huntc — COMMENTED** [review]
Did we get rid of the cargo fmt check?

**Comment by Dirbaio** [comment]
it's inside `ci.sh` now. I'll also try consolidating the other jobs too.

File: .github/workflows/rust.yml [hunk 1]
```diff
@@ -2,172 +2,36 @@ name: Rust
...
-    steps:
-      - uses: actions/checkout@v2
-      - uses: actions-rs/toolchain@v1
-        with:
-          toolchain: stable
-      - name: Check fmt
-        run: find -name '*.rs' | xargs rustfmt --check  --skip-children --unstable-features --edition 2018
+          curl -L -o /usr/local/bin/cargo-batch https://github.com/embassy-rs/cargo-batch/releases/download/batch-0.1.0/cargo-batch
+          chmod +x /usr/local/bin/cargo-batch
+          ./ci.sh
```
Comment (huntc) [inline .github/workflows/rust.yml:29]: What are we pulling down here? May be a comment would help describe?

**Comment by Dirbaio** [comment]
it's not really faster :'(

**Pull request merged**